docs/compute.md

```diff
@@ -309,6 +309,24 @@ With this simple but first initial algorithm, we managed to mitigate automatical
 
 You have more than 2 hosts? Again, same idea, it will also work.
 
+## VM anti-affinity
+
+VM anti-affinity is a feature that prevents VMs with the same user tags to run on the same host. This functionality is available directly in the load-balancer plugin.
+This way, you will avoid having pairs of redundant VMs or similar put on the same host.
+
+Let's see a simple example: you have multiple VMs using MySQL and Postgresql with high availability/replication. Obviously, you don't want to lose the replicated database inside VMs on the same physical host. Just create your plan like this:
+
+![](https://xen-orchestra.com/blog/content/images/2021/03/antiaffinity.png)
+
+- Simple plan: means no active load balancing mechanism here
+- Anti-affinity: we added our 2x tags, meaning any VMs with one of these tags will never run on the same host (as possible) with another VM having the same tag
+
+You can also use the performance plan with the anti-affinity mode activated to continue to migrate non-tagged VMs.
```

Comment:
Similarly, as we're on XCP-ng docs, I don't know what the performance plan is. Add a link to XO docs?

Reply:
I think we must be coherent and bring features logically (you shouldn't discover a term/feature without any context before on what it does)